client/src/pages/Explorer/Explore.js

```diff
@@ -134,36 +148,58 @@ const Explore = () => {
     });
   }
 
+  const pathName = window.location.pathname;
+  const smSpacing = pathName === "/profile/favoritedestinations" ? 6 : 3;
+  const mdSpacing = pathName === "/profile/favoritedestinations" ? 4 : 3;
   return (
     <>
       <Container className={classes.pageContainer}>
-        <Typography variant="h4" className={classes.title}>
-          Explore destinations
-        </Typography>
-        <Typography
-          variant="subtitle2"
-          className={(classes.title, classes.subtitle2)}
-        >
-          World's top destinations to explore
-        </Typography>
+        {pathName === "/profile/favoritedestinations" ? (
+          <Grid container justify="space-between">
+            <Typography
+              variant="h4"
+              style={{ fontWeight: 600 }}
+              className={classes.title}
+            >
+              Favorite Destinations
+            </Typography>
+            <Button variant="outlined">
+              <Link className={classes.exploreLink} to="/explore">
+                Explore
+              </Link>
+            </Button>
+          </Grid>
+        ) : (
+          <>
+            <Typography variant="h4" className={classes.title}>
+              Explore destinations
+            </Typography>
+            <Typography
+              variant="subtitle2"
+              className={(classes.title, classes.subtitle2)}
+            >
+              World's top destinations to explore
+            </Typography>
 
-        <Button
-          style={{ backgroundColor: "#fff" }}
-          className={classes.shuffle}
-          onClick={handleShuffleButton}
-        >
-          <Tooltip title="Shuffle Cities">
-            <ShuffleIcon />
-          </Tooltip>
-        </Button>
+            <Button
+              style={{ backgroundColor: "#fff" }}
+              className={classes.shuffle}
+              onClick={handleShuffleButton}
+            >
+              <Tooltip title="Shuffle Cities">
+                <ShuffleIcon />
+              </Tooltip>
+            </Button>
+          </>
+        )}
         <Grid
           container
           spacing={3}
           justify="center"
           className={classes.gridContainer}
         >
-          {places?.map((place, ind) => (
-            <Grid item key={place.name} xs={12} sm={3}>
+          {places?.map((place, _) => (
```

Comment:
If we don't need the index you just don't need to define it at all. `places.map(place => ...)`
Also I think what we're looking for here is `places && places.length > 0 && places.map`